src/ophys_etl/workflows/db/schemas.py

```diff
@@ -79,6 +79,13 @@ class OphysROI(SQLModel, table=True):
     y: int
     width: int
     height: int
+    is_in_motion_border: Optional[bool] = False
+    is_small_size: Optional[bool] = False
+    is_decrosstalk_invalid_raw: Optional[bool] = False
+    is_decrosstalk_invalid_raw_active: Optional[bool] = False
+    is_decrosstalk_invalid_unmixed: Optional[bool] = False
+    is_decrosstalk_invalid_unmixed_active: Optional[bool] = False
+    is_decrosstalk_ghost: Optional[bool] = False
```

Comment:
by setting the default value to `False`, there will be a state in which the value may incorrectly be `False` e.g. before decrosstalk has run. I would recommend to set instead to `None` which would indicate that it hasn't been run yet, or that it doesn't apply. Or leave the decrosstalk related flags out and they can be added to a separate table once decrosstalk runs.